lib/tasks/add_transcripts_to_oral_histories.rake

```diff
@@ -0,0 +1,57 @@
+namespace :scihist do
+  desc """
+    Goes through all the oral histories and adds transcripts from a file on the disk, for those missing them:
+
+    bundle exec rake scihist:add_transcripts_to_oral_histories
+  """
+
+  task :add_transcripts_to_oral_histories => :environment do
+    files_location = '/tmp/ohms_transcript_files/'
+    progress_bar = ProgressBar.create(total: Work.where("json_attributes -> 'genre' ?  'Oral histories'").count, format: "%a %t: |%B| %R/s %c/%u %p%% %e")
+    Work.where("json_attributes -> 'genre' ?  'Oral histories'").find_each(batch_size: 10) do |w|
+
+      if w.oral_history_content!&.searchable_transcript_source.present?
+        progress_bar.log("INFO: #{w.title} already has a searchable transcript.")
```

Comment:
If you're running this more than once, you're going to get a LOT of these on second run -- is it going to make it hard to see any actual errors you want to know about, crowding them out of log and making it easy to miss them?

Reply:
Correct!